.clj-kondo/com.gfredericks/test.chuck/clj_kondo/com/gfredericks/test/chuck/checking.clj

```diff
@@ -0,0 +1,19 @@
+(ns clj-kondo.com.gfredericks.test.chuck.checking
```

Comment:
Not really part of this PR per se but I updated the Kondo imports from our libraries.

As discussed elsewhere for the time being we are checking library Kondo configs in to our repo because the CI Kondo linter doesn't have a good to import them at runtime